src/public/javascripts/app.js

```diff
@@ -43,12 +44,13 @@ var defaultOptions = {
   animation: false
 };
 
-var createChart = function (ctx, dataset) {
+var createChart = function (ctx, dataset, color) {
```

Comment:
I am not sure but it looks like this function is never called w/ a color parameter; as the `statusCodesColors` is used in order to set the `borderColor` property of the `dataset` but later in line number 103